geomstats/information_geometry/statistical_metric.py

```diff
@@ -0,0 +1,49 @@
+"""Statistical metric to equip a statistical manifold."""
+
+import geomstats.backend as gs
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class StatisticalMetric(RiemannianMetric):
+    """
+    Defines statistical metric and connection induced from a divergence.
+
+    Uses definitions provided in Nielson's An Elementary Introduction to
+    Information Geometry, Theorem 4 on page 15 (https://arxiv.org/abs/1808.08271)
+    """
+
+    def __init__(self, dim, divergence):
+        self.dim = dim
+        self.divergence = self._unpack_tensor(divergence)
+
+    def _unpack_tensor(self, func):
+        def wrapper(tensor):
+            return func(tensor[..., : self.dim], tensor[..., self.dim :])
+
+        return wrapper
+
+    def metric_matrix(self, base_point):
```

Comment:
docstring missing parameter descriptions

Reply:
Give reference of the paper you mention in each docstring, using

References
-----------

(see in other places in geomstats for how References is used)

Reply:
Also, instead of giving the page where we should look for the equation, just write the equation in the docstring, and start the docstring by r"""

--> on the website, the docstring will show the equation in latex!

Eg. https://geomstats.github.io/api/geomstats.information_geometry.html#geomstats.information_geometry.fisher_rao_metric.FisherRaoMetric

Reply:
Can the docstring also explain the difference between this metric and the Fisher-Rao metric?

And maybe add a section in the docstring:

See Also
--------
FisherRaoMetric